### DISCUSSION TITLE
Use Deye SUN600G3-EU-230 without internet connection

### DISCUSSION THREAD

**Discussion by hajo62** [StephanJoubert/home_assistant_solarman#151] [Q&A]
Hi,

I wonder how I could use the Deye SUN600G3-EU-230 without giving it internet access?

If I disable internet access for the inverter, the daily production isn't resetted during the night. When the inverter starts in the morning, it still reports (and accumulates to) the value from yesterday. When I enable internet connection for some minutes, the value is resetted.
Does the inverter need a connection to the Chinese servers or to a time server or ... ?

I just have two panels and a single converter. 
I assume the `grid current` and `grid voltage` B and C, as well as `Output Apparent Power` and `Output Reactive Power` and `Total Output AC Power` are always 0 and aren't relevant for me?

**Comment by jedie** [comment]
I have the same Question: How to setup the Home assistant integration without giving Deye internet access?
